HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -1904,6 +1906,34 @@ def from_doc(building)
     end
   end
 
+  class Schedules < BaseElement
```

Comment:
The name 'Schedules' is too specific, I suspect we will end up with many non-schedule inputs in here. What do you think about calling this 'BuildingHeader' instead? So we will have an HPXML Header and then one or more Building Headers.